WcaOnRails/app/jobs/sync_mailing_lists_job.rb

```diff
@@ -11,6 +11,7 @@ def perform
     GsuiteMailingLists.sync_group("communication@worldcubeassociation.org", Team.wct.current_members.includes(:user).map(&:user).map(&:email))
     GsuiteMailingLists.sync_group("competitions@worldcubeassociation.org", Team.wcat.current_members.includes(:user).map(&:user).map(&:email))
     GsuiteMailingLists.sync_group("disciplinary@worldcubeassociation.org", Team.wdc.current_members.includes(:user).map(&:user).map(&:email))
+    GsuiteMailingLists.sync_group("dataprotection@worldcubeassociation.org", Team.wdpc.current_members.includes(:user).map(&:user).map(&:email))
```

Comment:
Is this our first email address with two words in it? How about `data.protection@` instead?

Reply:
I believe so, because in other cases (e.g. Quality Assurance, Regional Organizations) we ended up using one word only. The @thewca/board discussed between `dataprotection@` and `data-protection@`, and decided to go with the first one. I don't feel too strongly about this, but maybe others do!

Reply:
I'm not a fan of having a period or hyphen in the email address. If we want just one word, how about privacy@?

Reply:
Why do you dislike the period, @cubewhiz? It's possible that google groups will actually direct emails to `data.protection@` and `dataprotection@` to the same place (or maybe that feature only exists for personal email, not groups).

I like `privacy@`, too.

Reply:
Just aesthetically, I think it's ugly.  @thewca/board how do you feel about privacy@?

Reply:
Yeah privacy is "ok" though I opted against it before but happy to go with it if that is the consensus.

Reply:
Uglier than jamming two words together without any visual separator?

Reply:
yes

Reply:
I would prefer to use any of the variations of `dataprotection@` over `privacy@`, although adding `privacy@` as an alias sounds like a good idea!

Reply:
I think `data.privacy` looks nicer than `dataprivacy`, but @cubewhiz feels the other way. @AlbertoPdRF, do you want to make an executive decision or put it up to a vote? Whatever you prefer.

Reply:
For the record, I did try emailing `soft.ware@worldcubeassociation.org` to see if it would work. It did not.

![image](https://user-images.githubusercontent.com/277474/51508201-217c8d00-1da9-11e9-94a9-dc2902433381.png)